connectors/sources/mysql.py

```diff
@@ -106,11 +99,44 @@ async def close(self):
         await self.connection_pool.wait_closed()
         self.connection_pool = None
 
+    def _validate_connection_fields(self):
+        """Validates whether user input is empty or not for connection string
+
+        Raises:
+            Exception: Configured keys can't be empty
+        """
+        connection_fields = ["host", "port", "user", "password", "database"]
+        empty_connection_fields = []
+        for field in connection_fields:
+            if not self.configuration[field]:
+                empty_connection_fields.append(field)
+
+        if empty_connection_fields:
+            raise Exception(
+                f"Configured keys: {empty_connection_fields} can't be empty."
+            )
+
+        if (
+            isinstance(self.configuration["port"], str)
+            and not self.configuration["port"].isnumeric()
+        ):
+            raise Exception("Configured port can't include string literal.")
```

Comment:
this message is cryptic. what about `Configured port has to be an integer`

Reply:
Done